src/mpl2/src/SACoreSoftMacro.cpp

```diff
@@ -138,8 +138,9 @@ float SACoreSoftMacro::getNormNotchPenalty() const
 float SACoreSoftMacro::calNormCost() const
 {
   float cost = 0.0;  // Initialize cost
+  const float outline_area = outline_height_ * outline_width_;
   if (norm_area_penalty_ > 0.0) {
-    cost += area_weight_ * (width_ * height_) / norm_area_penalty_;
+    cost += area_weight_ * (width_ * height_) / outline_area;
   }
```

Comment:
Explain this change

Reply:
area penalty is normalized by the floorplan outline area